packages/contentful--app-scripts/package.json

```diff
@@ -61,5 +62,16 @@
     "open": "8.4.2",
     "ora": "5.4.1"
   },
-  "gitHead": "4c3506be3f52c7a8aae17deaa75acefc9a805b42"
+  "gitHead": "4c3506be3f52c7a8aae17deaa75acefc9a805b42",
+  "devDependencies": {
+    "@tsconfig/node16": "^16.1.0",
```

Comment:
I think we are still supporting `node14`, right? So we should use that as tsconfig